src/autocomplete/autocomplete.ts

```diff
@@ -181,3 +183,26 @@ export function parseClickHouseQuery(
         cursor,
     );
 }
+export function parseYQLQueryWithoutCursor(query: string): Pick<YQLAutocompleteResult, 'errors'> {
```

Comment:
Please add a newline on top. Or help us to setup linter so that it will autofix it

Reply:
Ok, I'll add a rule to eslinrc